### PR TITLE
Makes a new Holy crate to heal deal with cults - And null rod

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

New crate has a "null rod" type weapon, just for the holy/anti-magic as well as ruin clearing
Some holy water, candles and hoods - Costs 8k 

New REALY weak null rod for cargo crates - Maybe throw this into some leagon one day
simple deals 6 harm and 4 throw. Thus not ment for normal combat, its ment to protect you against the veil of blood or the corrupted light.

## Why It's Good For The Game

Ruins are hard as hell to clear without a null rod, and magic seems to just relies on hydro making holy melons, this should give cargo more to buy then just mind-shields and more ways to counter ruins/magic spells with a weak weapon*

## Changelog
:cl:
add: crate/nullrod
/:cl:

### CODE REVIEW

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -707,3 +704,14 @@
 	righthand_file = 'icons/mob/inhands/weapons/staves_righthand.dmi'
 	w_class = WEIGHT_CLASS_NORMAL
 	attack_verb = list("bashes", "smacks", "whacks")
+
+/obj/item/nullrod/silver_shortblade //Its not ment for combat.
+	name = "eloquent silver blade"
+	desc = "A dull blade made of silver, has a few rosary beads imbedded into the handle. Seems to be ment for pray rather then melee..."
```
Comment (CameronWoof) [inline code/game/objects/items/holy_weapons.dm]: embedded, not imbedded. _meant_ for pray_er_ rather _than_

**Comment by Poojawa** [comment]
Cargo powercreep by Lilly again.

Why are you nerfing the Chaplain's role?

**Comment by Trilbyspaceclone** [comment]
@Poojawa Cult win rates. + good RP for people that get the cult followers box

**Comment by Poojawa** [comment]
"Good RP for people that get the cult followers box"

No, it's blatant powercreep then.

**Comment by silicons** [comment]
people underestimate how much damage even one robust guy with a nullrod can do to a cult when it nullifies most cult magic. i don't think we need more of this unless we rework the whole "holy combat/protection" system.

**Pull request closed without merging**